src/clientdlg.cpp

```diff
@@ -1001,6 +1000,7 @@ void CClientDlg::ShowConnectionSetupDialog()
     // show connect dialog
     bConnectDlgWasShown = true;
     ConnectDlg.show();
+    ConnectDlg.setWindowTitle( MakeClientNameTitle( tr ( "Connect" ) , pClient->strClientName ) );
```

Comment:
> Sorry, spaces after the method name and before the `(`, too, please

You've removed the one that was before the `(`.  It needs a space both sides.  (Yes, I phrased it a bit clumsily now I read it back.)